src/View/Form/FormContext.php

```diff
@@ -167,6 +166,24 @@ public function getRequiredMessage($field)
         return null;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getMaxLength($field)
+    {
+        $validator = $this->_form->getValidator();
+        if (!$validator->hasField($field)) {
+            return null;
+        }
+        foreach ($validator->field($field)->rules() as $rule) {
+            if ($rule->get('rule') === 'maxLength') {
+                return isset($rule->get('pass')[0])? $rule->get('pass')[0] : null;
```

Comment:
The `isset()` check needs to be removed here too.

Reply:
thanx,  i have removed